docs/topics/signals.txt

```diff
@@ -12,30 +12,15 @@ has taken place. They're especially useful when many pieces of code may be
 interested in the same events.
 
 Django provides a :doc:`set of built-in signals </ref/signals>` that let user
-code get notified by Django itself of certain actions. These include some useful
-notifications:
+code get notified by Django itself of certain actions.
 
-* :data:`django.db.models.signals.pre_save` &
-  :data:`django.db.models.signals.post_save`
+This is an example of the ``setting_changed`` signal, which is used by many pluggable
+Django apps::
 
-  Sent before or after a model's :meth:`~django.db.models.Model.save` method
-  is called.
-
-* :data:`django.db.models.signals.pre_delete` &
-  :data:`django.db.models.signals.post_delete`
-
-  Sent before or after a model's :meth:`~django.db.models.Model.delete`
-  method or queryset's :meth:`~django.db.models.query.QuerySet.delete`
-  method is called.
-
-* :data:`django.db.models.signals.m2m_changed`
-
-  Sent when a :class:`~django.db.models.ManyToManyField` on a model is changed.
-
-* :data:`django.core.signals.request_started` &
-  :data:`django.core.signals.request_finished`
-
-  Sent when Django starts or finishes an HTTP request.
```

Comment:
I think we could still include a sentence about some examples of signals, and join the "See the built-in signal documentation..." sentenece in the same paragraph.

Reply:
I'm not exactly sure this is what you meant, but I've updated it. I'll update the screenshots in the PR description